### ISSUE TITLE
bones are renamed

### ISSUE THREAD

**Issue by KatekovAnton** [gregeryb/DAEBlend#12]
Hi. We were trying to use this plugin to export dae but it seems it renamed all bones to "id-bone-<number>". Is it intentional? is it possible to keep original bones names?

**Comment by KatekovAnton** [comment]
I was looking at wrong field

**Comment by gregdavisd** [comment]
It puts the node names in the <skin> controller as a <Name_array> which are just as they are found in blender. But that's just the rest position (inverted matrices), the pose positions is then defined as a node tree and referenced with <skeleton>. If there's a name collision in the node tree then it will generate a unique id with a number on the end. It's all very complicated. I found that DAE loaders interpret the format in wildly different ways so DAE is extremely hard to work with. And I found that it most cases the intended is to use a JS library to read it like Three.JS. Which means it makes more sense just to export from Blender in JSON format directly. Maybe there's a good JS Blender exporter out there.

**Comment by gregdavisd** [comment]
It puts the node names in the "skin" controller as a "Name_array" which are just as they are found in blender. But that's just the rest position (inverted matrices), the pose positions is then defined as a node tree and referenced with "skeleton". If there's a name collision in the node tree then it will generate a unique id with a number on the end. It's all very complicated. I found that DAE loaders interpret the format in wildly different ways so DAE is extremely hard to work with. And I found that it most cases the intended use is a JS library to read it. Like Three.JS. Which means it makes more sense just to export from Blender in JSON format directly. Maybe there's a good JS Blender exporter out there.
